### ISSUE TITLE
Meadow version check thinks 0.6.2.0 is less than 0.6.1.0

### ISSUE THREAD

**Issue by NevynUK** [WildernessLabs/Meadow.CLI#142]
Just received the following message when upgrading a board:

```
Meadow StdInfo: Mono will not start. Mono version is not available (Meadow.OS version 0.6.2.0).
Your OS version is older than 0.6.1.0. A bulk flash erase is highly recommended.
Proceed? (Y/N) Press Y to erase flash, N to continue install without erasing
```

Upgrading to 0.6.3.0 using CLI 0.19.1.

**Comment by adrianstevens** [comment]
Fixed with this commit: 
https://github.com/WildernessLabs/Meadow.CLI/commit/ae12add13c32415179c04f157f2218510092d627

Specifically changed the wording to:
Your OS or runtime version is older than {MINIMUM_OS_VERSION}. A flash erase is highly recommended.

**Comment by NevynUK** [comment]
Closed following beta 6.4 release.
